### PR TITLE
[IMP] Inventory: Updating for 16.0 no save button

### PR DESCRIPTION
**Pull request by larm-odoo**
No save button in 16, made a couple changes in 2 places so document is correct for 16.

### CODE REVIEW

File: content/applications/inventory_and_mrp/inventory/shipping/operation/label_type.rst [hunk 1]
```diff
@@ -13,15 +13,15 @@ Configuration
...
+Shipping Methods.` Click on a delivery method to choose it. For the following example,
 *FedEx International* will be used.
 
 .. image:: label_type/shipping-options.png
    :align: center
    :alt: Different shipping methods.
 
 In the :guilabel:`Configuration` tab, under :guilabel:`Label Type`, choose one of the label types
-available. The availability will vary depending on the carrier. :guilabel:`Save` the changes.
+available. The availability will vary depending on the carrier.
```
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/inventory/shipping/operation/label_type.rst:24]:
```suggestion
available. The availability varies depending on the carrier.
```

**Review by ksc-odoo — COMMENTED** [review]
Hey there @larm-odoo ! Great job on this minor adjustment! Only had one super-small suggestion to maintain the present tense, but once that's done, I'd say it's ready to move on to the next stage 👍

**Comment by larm-odoo** [comment]
Thanks @ksc-odoo! @samueljlieber, this is ready for you!

File: content/applications/inventory_and_mrp/inventory/shipping/operation/label_type.rst [hunk 2]
```diff
@@ -13,15 +13,15 @@ Configuration
 =============
 
 In the :menuselection:`Inventory` module, go to :menuselection:`Configuration --> Delivery -->
-Shipping Methods.` Choose a delivery method and click :guilabel:`Edit`. For the following example,
+Shipping Methods.` Click on a delivery method to choose it. For the following example,
 *FedEx International* will be used.
```
Comment (samueljlieber) [inline content/applications/inventory_and_mrp/inventory/shipping/operation/label_type.rst:17]: Early line break
```suggestion
Shipping Methods.` Click on a delivery method to choose it. For the following example, *FedEx
International* will be used.
```

**Review by samueljlieber — APPROVED** [review]
Hi @larm-odoo! Nice quick edits 🙂 I caught a single early line break, please correct this and then you can pass this PR on to final review - approving now!

**Comment by larm-odoo** [comment]
Ready for you, @StraubCreative! Just 2 small changes to this doc to make it accurate for 16.

**Review by StraubCreative — APPROVED** [review]
Looks good @larm-odoo .

Note for feature branch naming convention:
This should be in all lowercase, and the short description should be more specific, ideally naming the change as well as the file being changed. "Making targeted edits" doesn't quite say anything about the feature(s) being added to the code base 😉 

A more ideal name for this branch might have been:
`16.0-inventory-update-label-type-larm`

@robodoo r+

**Pull request closed without merging**